src/data/real/ereal.lean

```diff
@@ -49,10 +49,11 @@ also do some limits stuff (liminf/limsup etc).
 See https://isabelle.in.tum.de/dist/library/HOL/HOL-Library/Extended_Real.html
 -/
 
+open function
 open_locale ennreal nnreal
 
 /-- ereal : The type `[-∞, ∞]` -/
-@[derive [has_top, comm_monoid_with_zero,
+@[derive [has_top, comm_monoid_with_zero, nontrivial,
```

Comment:
What motivates this?

Reply:
`example : nontrivial ereal := infer_instance` fails without it, and it's needed for `bot_lt_top`